cpp/tests/io/arrow_io_source_test.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_utilities.hpp>
+#include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/cudf_gtest.hpp>
+#include <cudf_test/table_utilities.hpp>
+#include <cudf_test/type_lists.hpp>
+
+#include <cudf/io/datasource.hpp>
+#include <cudf/io/parquet.hpp>
+
+#include <arrow/io/api.h>
+
+#include <memory>
+#include <string>
+
+// Global environment for temporary files
+auto const temp_env = static_cast<cudf::test::TempDirTestEnvironment*>(
+  ::testing::AddGlobalTestEnvironment(new cudf::test::TempDirTestEnvironment));
+
+// Base test fixture for tests
+struct ArrowIOTest : public cudf::test::BaseFixture {
+};
+
+TEST_F(ArrowIOTest, S3Filesystem)
+{
+  std::string s3_uri = "s3://ursa-labs-taxi-data/2010/06/data.parquet?region=us-east-2";
```

Comment:
Could this be updated to use a smaller file preferably hosted by rapids instead of a public dataset.

Reply:
That would be great! Do we have an existing parquet file in our rapids public dataset? If so could you point me to that?

Reply:
https://github.com/rapidsai/ops/issues/1630